CHANGELOG.md

```diff
@@ -72,6 +87,19 @@ Use BlobStore cache instead.
 - JAMES-3204 Push limit to Cassandra backend when reading messages - before that message listing queries where always reading at least 5000 rows, and triggering other reads for these rows.
 - JAMES-3201 ReIndexing enhancements
 - JAMES-3179 Fix UpdatableTickingClock thread safety issue
+- MAILBOX-405 Renaming too much mailboxes at once was failing on top of the Cassandra mailbox
+- JAMES-3513 Wrong UID dispatched on the EventBus for StoreMessageIdManager::setInMailboxes
+- JAMES-3512 DigestUtil: close base64 encoding stream
+- JAMES-3487 Allow setting on*Exception parameters for Bounce
+- JAMES-3511 Solve java.util.NoSuchElementException: heartbeatHandler
+- JAMES-3507 Fix broken IMAP APPEND literalSizeLimit option preventing from buffering large requests to files
+- JAMES-3438 des-ambiguity error message for Email/set create Content-Transfer-Encoding rejection
+- JAMES-3477 Fix NPE when concurrently updating MimeMessage
```

Comment:
it may be worth mentionning that this could have impacts on the server's performance profile...